packages/js/src/generators/release-version/utils/resolve-version-spec.spec.ts

```diff
@@ -67,7 +67,7 @@ describe('resolveVersionSpec()', () => {
         'file:../projectB',
         '/packages/projectB'
       )
-    ).toEqual(expect.stringContaining(join('/packages/projectB')));
+    ).toEqual(expect.stringContaining(joinPathFragments('/packages/projectB')));
```

Comment:
The `resolveVersionSpec` function always returns a POSIX formatted path, so we don't need to use a util to format a static expectation. We should directly expect the path in the desired format.
```suggestion
    ).toEqual(expect.stringContaining('/packages/projectB'));
```

Reply:
it error
```
 FAIL   js  packages/js/src/generators/release-version/utils/resolve-version-spec.spec.ts
  resolveVersionSpec()
    √ should work for specific name and spec (4 ms)                                                                                                                                                                              
    √ should work for a workspace spec                                                                                                                                                                                           
    × should for a file reference (1 ms)                                                                                                                                                                                         
    × should work for a yarn classic style link reference (1 ms)                                                                                                                                                                 
    with a workspace alias                                                                                                                                                                                                       
      √ should work for a * workspace alias (1 ms)                                                                                                                                                                               
      √ should work for a ^ workspace alias                                                                                                                                                                                      
      √ should work for a ~ workspace alias (1 ms)                                                                                                                                                                               
                                                                                                                                                                                                                                 
  ● resolveVersionSpec() › should for a file reference                                                                                                                                                                           
                                                                                                                                                                                                                                 
    expect(received).toEqual(expected) // deep equality

    Expected: StringContaining "\\packages\\projectB"
    Received: "/packages/projectB"

      68 |         '/packages/projectB'
      69 |       )
    > 70 |     ).toEqual(expect.stringContaining(join('/packages/projectB')));
         |       ^
      71 |   });
      72 |
      73 |   it('should work for a yarn classic style link reference', async () => {

      at Object.<anonymous> (src/generators/release-version/utils/resolve-version-spec.spec.ts:70:7)

  ● resolveVersionSpec() › should work for a yarn classic style link reference

    expect(received).toEqual(expected) // deep equality

    Expected: StringContaining "\\packages\\projectB"
    Received: "/packages/projectB"

      79 |         '/packages/projectB'
      80 |       )
    > 81 |     ).toEqual(expect.stringContaining(join('/packages/projectB')));
         |       ^
      82 |   });
      83 | });
      84 |

      at Object.<anonymous> (src/generators/release-version/utils/resolve-version-spec.spec.ts:81:7)

Test Suites: 1 failed, 1 total                                                                                                                                                                                                   
Tests:       2 failed, 5 passed, 7 total                                                                                                                                                                                         
Snapshots:   0 total
Time:        3.24 s
Ran all test suites matching /src\\generators\\release-version\\utils\\resolve-version-spec.spec.ts/i.
```

Reply:
The output should always be `/packages/projectB`. As you can see in the change I suggested, that's the expectation, without using `join` or `joinPathFragments`.

Reply:
okay sorry, I misunderstood